Azure/cc-setup-azure.sh

```diff
@@ -0,0 +1,169 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Description: This script will launch a VM using Azure API, and install
+# all components needed for Clear Containers.
+
+MYSELF="${0##*/}"
+
+# Azure/VM configuration
+RESOURCE_GROUP="cc-metrics"
+VM_NAME="vm-metrics"
+OS_SYSTEM="UbuntuLTS"
+VM_SIZE="Standard_D2s_v3"
+CLOUD_INIT_FILE="cloud-init.txt"
+VM_ADMIN_USER="adminazure"
+GRP_LOCATION="eastus"
+
+# Disks configuration
+DISK_NAME="devicemapper"
```

Comment:
can the disk names be a combo of the vm name + disk so that people can do multiple of these per resource group

Reply:
@jessfraz  could be good idea, lets do it :)